R/transformCounts.R

```diff
@@ -211,6 +211,7 @@ setMethod("transformSamples", signature = c(x = "SummarizedExperiment"),
         # Call general transformation function with MARGIN specified
         x <- transformAssay(x = x, assay.type = assay.type,
```

Comment:
In examples we use `se` instead of `x` as we like to highlight the fact that these methods work apply to SE objects.

Can you still update the manpage examples with that?  Just changing x to tse. No need for other changes than the examples.